spring-integration-feed/src/test/java/org/springframework/integration/feed/inbound/FeedEntryMessageSourceTests.java

```diff
@@ -38,13 +38,13 @@
  * @author Oleg Zhurakousky
  * @author Mark Fisher
  * @author Gary Russell
+ * @author Aaron Loes
```

Comment:
Your name to the `FeedEntryMessageSource` as well, with update for Copyright to the current year